interface/forms/observation/new.php

```diff
@@ -39,138 +39,27 @@
 
 $check_res = $formid ? $check_res : array();
 
+$reasonCodeStatii = ReasonStatusCodes::getCodesWithDescriptions();
+$reasonCodeStatii[ReasonStatusCodes::EMPTY]['description'] = xl("Select a status code");
+
 ?>
 <html>
 <head>
     <title><?php echo xlt("Observation"); ?></title>
 
-    <?php Header::setupHeader(['datetime-picker']); ?>
+    <?php Header::setupHeader(['datetime-picker', 'reason-code-widget']); ?>
 
+    <!--Update the version (v flag) to cache bust if you modify this file -->
+    <script src="<?php echo attr($GLOBALS['webroot']); ?>/interface/forms/observation/observation.js?v=1" type="text/javascript"></script>
```

Comment:
substitute the 1 with `$GLOBALS['v_js_includes']`. This gets set in version.php and is used for cache bust. We basically increment it manually whenever needed. Note there is a cool mechanism for developers where this will basically be a random number if have a .env file with the following:
```
# Possible options are prod or dev. No spaces, no quotes.
OPENEMR__ENVIRONMENT=dev
```